### DISCUSSION TITLE
More topics to be discussed in the mentor meetup sessions

### DISCUSSION THREAD

**Discussion by RehabAbotalep** [MohamedRadwan-DevOps/devops-step-by-step#589] [Ideas]
More topics to be discussed in the mentor meetup sessions:

1. Sell and negotiation
2. Convey what you think and feel
3. Shut up, listen and learn from others
4. Adapt, impressive and overcome obstacles
6. Make decisions based on facts not based on emotions
7. Speak in front of a large audience
8. Keep trying even after failure
9. Invest money on own
10. Do things irrespective of situations
11. Self-analysis
12. Learn how to learn
13. Understand what others feel
14. Remain consistent
16. Write words to persuade and influence others
17. Ask for help

Any suggestions?

**Comment by ShereenS** [comment]
The work you do does not need to be the passion you live for.

**Reply by naja-ahmed** [reply]
How and when to say no

**Comment by Mohamedalifarouk** [comment]
Time management

**Comment by naja-ahmed** [comment]
Increase self-confidence

**Comment by naja-ahmed** [comment]
Dealing with toxic people without being affected psychologically

**Comment by naja-ahmed** [comment]
How to be dynamic
